apps/website/src/components/layout/navbar/DesktopMenu.tsx

```diff
@@ -0,0 +1,194 @@
+              {"link" in link ? (
+                <NavLink href={link.link}>{link.title}</NavLink>
+              ) : (
+                <Menu as="div" className="relative">
+                  {({ open }) => (
+                    <>
+                      <Menu.Button
+                        className={[
+                          navLinkClassesMain,
+                          "flex items-center gap-2",
+                        ]
+                          .filter(Boolean)
+                          .join(" ")}
+                      >
+                        {link.title}
+                        <IconAngleDown
+                          size={16}
+                          className={`${
+                            open ? "translate-y-1" : "translate-y-0.5"
+                          } transition-transform`}
+                        />
+                      </Menu.Button>
+
+                      <DropdownMenuItems>
+                        {Object.entries(link.dropdown).map(([key, link]) => (
+                          <Menu.Item as="li" key={key}>
+                            {({ close }) => (
+                              <NavLinkSub
+                                href={link.link}
+                                className="w-full min-w-max"
+                                onClick={close}
+                              >
+                                {link.title}
+                              </NavLinkSub>
+                            )}
+                          </Menu.Item>
+                        ))}
+                      </DropdownMenuItems>
+                    </>
+                  )}
+                </Menu>
+              )}
+            </li>
+          ))}
+
+          {/* Notifications toggle */}
+          {/*<li>*/}
+          {/*  <NotificationsButton />*/}
+          {/*</li>*/}
+
+          {/* User menu */}
+          <li>
+            {sessionData ? (
+              <Menu as="div" className="relative flex h-full items-center">
+                <Menu.Button
+                  as="button"
+                  className="mx-4 cursor-pointer select-none appearance-none rounded-full border-2 border-transparent transition-colors hover:border-white aria-expanded:border-white"
+                >
+                  <span className="sr-only">Open user menu</span>
+                  <ProfileInfoImage />
+                </Menu.Button>
+
+                <DropdownMenuItems>
+                  <Menu.Item disabled>
+                    <div className="px-5 py-3">
+                      <ProfileInfo full />
+                    </div>
+                  </Menu.Item>
+
+                  <Menu.Item disabled>
+                    <div className="border-t opacity-30"></div>
+                  </Menu.Item>
+
+                  {showAdminLink && (
+                    <Menu.Item>
+                      <Link className="px-5 py-3" href="/admin/dashboard">
+                        Admin
+                      </Link>
```

Comment:
This should probably be `NavLinkSub`, as its missing hover states currently?

Reply:
yup, fixed